tests/modules/merqury/test.yml

```diff
@@ -0,0 +1,35 @@
+- name: merqury test_merqury
+  command: nextflow run ./tests/modules/merqury -entry test_merqury -c ./tests/config/nextflow.config  -c ./tests/modules/merqury/nextflow.config
+  tags:
+    - merqury
+  files:
+    - path: output/merqury/genome_only.bed
+      md5sum: b611f22cde0e410a2ca07c1eefd042d3
+    - path: output/merqury/genome_only.wig
+      md5sum: 19cf44989af72af597ef80d3489b4882
+    - path: output/merqury/test.completeness.stats
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
That seems the offending empty file md5sum? I do not know how that works now, but maybe removing the checksum and check only for file existence would solve the linting?